src/test/java/com/github/jscancella/conformance/profile/BagitProfileTest.java

```diff
@@ -0,0 +1,115 @@
+package com.github.jscancella.conformance.profile;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.HashMap;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class BagitProfileTest extends AbstractBagitProfileTest{
+  
+  @Test
+  public void testToString() throws Exception{
```

Comment:
This test doesn't add much value as there isn't any logic in `toString`. I would remove it, as it just makes for more work to maintain it.